README.md

```diff
@@ -245,6 +249,18 @@ List of routes prefix. The Perimeterx module will always match request uri by th
 ...
 ```
 
+#### <a name="whitelist-routes"></a> Whitelist Routes
+
+List of routes prefix. The Perimeterx module will skip detection if the prefix match request uri .
+
+**default: None**
+
+```xml
+...
+  routesWhitelist="/login,/user/profile"
+...
+```
+
```

Comment:
Please document the `blockSpecificRoutes` part as well.

Reply:
👍